### ISSUE TITLE
Analytics dashboard 

### ISSUE THREAD

**Issue by mosif16** [ArcadeLabsInc/workerbee#10]
Dashboard to show usage of platform and types of jobs completed by workers

**Comment by earonesty** [comment]
yes there's a bounty for this that a i believe someone is almost finished with.
